libs/sdk-ui-tests/backstop/run-backstop.sh

```diff
@@ -57,10 +57,6 @@ docker network create "${BACKSTOP_NET}" || { echo "Network creation failed" && e
 
     echo "nginx with storybook is up"
 
-    echo "Cleaning up the backstop output directory."
```

Comment:
Why was this step removed?

Reply:
Base on Lupko hints. Approve command was not working (all generated screens where removed before moved into reference dir)